etc/make_release.py

```diff
@@ -650,7 +654,9 @@ def create_github_release_draft(gh_repo,
     gh_release = gh_repo.create_git_release(tag=release_tag, name=release_name,
```

Comment:
Suggest modifying the `generate_release_notes` function to document signature verification:

```python
    footer = textwrap.dedent("""
    ## Feedback
    To report a bug or request a feature, please open a ticket in the MongoDB issue management tool Jira:

    - [Create an account](https://jira.mongodb.org) and login.
    - Navigate to the [CXX project](https://jira.mongodb.org/browse/CXX)
    - Click `Create`.
                             
    ## Signature verification
    Use `gpg` to verify the signature. The public key for `cpp-driver` is available on https://pgp.mongodb.com/.
    """).lstrip()
```



This suggestion is intended to satisfy the requirement noted in DRIVERS-2895:

> Drivers MUST provide documentation for users to verify release artifacts if they wish